src/pages/volunteer.tsx

```diff
@@ -30,11 +31,32 @@ export default function volunteer(): JSX.Element {
         </div>
     );
 
+    const socialIcons: Icon[] = [
+        {
+            image: {
+                url: 'https://www.meetup.com/open-sgf/',
+                src: '/images/meetup-logo.svg',
+                alt: 'Meetup Logo',
+            },
+            title: <div className="h1">Meetup</div>,
+        },
+        {
+            image: {
+                url: 'https://discord.gg/jFD8dZP',
+                src: '/images/discord-logo.svg',
+                alt: 'Discord Logo',
+            },
+            title: <div className="h1">Discord</div>,
```

Comment:
Similar to profile avatars, maybe prefer `<figure><figcaption>` with inner  `<span className="h1">`  here.